src/endpoints/authentication/impl.cpp

```diff
@@ -246,10 +246,10 @@ namespace irods::http::handler
 					     irods::http::globals::oidc_configuration().at("redirect_uri").get_ref<const std::string&>()}};
 
 					// Encode the string, hit endpoint, get res
-					nlohmann::json res_item{hit_token_endpoint(encode_body(args))};
+					nlohmann::json oicd_response{hit_token_endpoint(encode_body(args))};
```

Comment:
oicd_response -> oidc_response

and then 3x below...

Reply:
Hopefully got the spelling right this time...